README.md

```diff
@@ -48,8 +48,8 @@
 	- OS
 		- [Operating Systems](#os)
 	- Gaming
-		- [Bindings](#game-bindings)
-		- [Game Engine Bindings](#game-engine-bindings)
+		- [Game and Graphics Bindings](#game-bindings)
+		- [Game Engines](#game-engines)
```

Comment:
I think you meant to link to `D Game Engines` here? It's below Game and Graphics Bindings at least

Reply:
RIght